File: initializer/android-ios/ios/ios/HomeScreen.swift
//
//  ContentView.swift
//  ios
//
//  Created by hyun kim on 06/10/20.
//

import SwiftUI
import base

func HomeScreen(_ model: HomeViewModel) -> some View {
    Screen(model) {
        Text("\(HelloIosKt.helloText) \(+model.world ?? "")")
    }
}

File: initializer/android-ios/android/src/main/java/kim/jeonghyeon/template/HomeScreen.kt
package kim.jeonghyeon.template

import androidx.compose.foundation.Text
import androidx.compose.runtime.Composable
import kim.jeonghyeon.androidlibrary.compose.Screen
import kim.jeonghyeon.androidlibrary.compose.unaryPlus
import kim.jeonghyeon.base.helloText
import kim.jeonghyeon.base.HomeViewModel

@Composable
fun HomeScreen(model: HomeViewModel) {
    Screen(model) {
        Text("$helloText ${+model.world}")
    }
}